### ISSUE TITLE
Skills incorrect and save unable to load

### ISSUE THREAD

**Issue by CorporalChrome** [RenegadeLucien/project-tenacity#70]
When going to complete the next task that was listed for me I noticed that it was a rather difficult task for my level as it said for me to complete "Aminishi? Completed it mate" then "brothers in more arms". When I looked at my skills in the skills tab it said that my ranged was 20 levels higher than its actual amount. I then correctly entered the XP and recalced and now my save no longer loads forcing me to delete it and create a new profile.

**Comment by RenegadeLucien** [comment]
Do you still have the save that didn't load? Did the extra Ranged XP come after completing a task or after creating and/or loading your profile?

**Comment by CorporalChrome** [comment]
No, I deleted the save without thinking about it, unfortunately. I noticed the ranged XP came after completing a task, I'm unsure of what task it was but it was from something basic. The change in XP wasn't a big deal as I could just adjust it to the correct amount and recalc.
